charts/flame-rollup/values.yaml

```diff
@@ -11,7 +11,7 @@ images:
     repo: ghcr.io/astriaorg/flame
     pullPolicy: IfNotPresent
     tag: sha-457e1f9
-    devTag: sha-457e1f9
+    devTag: latest
```

Comment:
We need to merge https://github.com/astriaorg/astria/pull/1972 for these changes to go into main.